client/src/app/modules/events/participant-list/invite-dialog/invite-dialog.component.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { IEventSchema } from './../../../../interfaces/event.interface';
+import { translateLoaderSpy } from './../../../../utils-test/test-utils.spec';
+import { NgbModal, NgbModule, NgbModalRef } from '@ng-bootstrap/ng-bootstrap';
+import { HttpClient } from '@angular/common/http';
+import { TranslateModule, TranslateLoader, TranslateService } from '@ngx-translate/core';
+import { TagInputModule } from 'ngx-chips';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { InviteDialogComponent } from './invite-dialog.component';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { HttpLoaderFactory } from '../../../../app.module';
+
+describe('InviteDialogComponent', () => {
+  let component: InviteDialogComponent;
+  let fixture: ComponentFixture<InviteDialogComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ InviteDialogComponent ],
+      imports: [
+        FormsModule,
```

Comment:
Later on, when I will have more time, we can discuss the stubmodules- they make tests faster & less dependant on other components/ modules